rasa/core/policies/keras_policy.py

```diff
@@ -170,41 +170,36 @@ def train(
         # noinspection PyPep8Naming
         shuffled_X, shuffled_y = training_data.shuffled_X_y()
 
-        self.graph = tf.Graph()
-        with self.graph.as_default():
-            # set random seed in tf
-            tf.set_random_seed(self.random_seed)
-            self.session = tf.compat.v1.Session(config=self._tf_config)
-
-            with self.session.as_default():
-                if self.model is None:
-                    self.model = self.model_architecture(
-                        shuffled_X.shape[1:], shuffled_y.shape[1:]
-                    )
-
-                logger.info(
-                    "Fitting model with {} total samples and a "
-                    "validation split of {}"
-                    "".format(training_data.num_examples(), self.validation_split)
-                )
+        tf.random.set_seed(self.random_seed)
 
-                # filter out kwargs that cannot be passed to fit
-                self._train_params = self._get_valid_params(
-                    self.model.fit, **self._train_params
-                )
+        if self.model is None:
+            self.model = self.model_architecture(
+                shuffled_X.shape[1:], shuffled_y.shape[1:]
+            )
 
-                self.model.fit(
-                    shuffled_X,
-                    shuffled_y,
-                    epochs=self.epochs,
-                    batch_size=self.batch_size,
-                    shuffle=False,
-                    verbose=obtain_verbosity(),
-                    **self._train_params,
-                )
-                # the default parameter for epochs in keras fit is 1
-                self.current_epoch = self.defaults.get("epochs", 1)
-                logger.info("Done fitting keras policy model")
+        logger.info(
+            "Fitting model with {} total samples and a "
+            "validation split of {}"
+            "".format(training_data.num_examples(), self.validation_split)
+        )
+
+        # filter out kwargs that cannot be passed to fit
+        self._train_params = self._get_valid_params(
+            self.model.fit, **self._train_params
+        )
+
+        self.model.fit(
+            shuffled_X,
+            shuffled_y,
+            epochs=self.epochs,
+            batch_size=self.batch_size,
+            shuffle=False,
+            verbose=obtain_verbosity(),
+            **self._train_params,
+        )
+        # the default parameter for epochs in keras fit is 1
+        self.current_epoch = self.defaults.get("epochs", 1)
+        logger.info("Done fitting keras policy model")
```

Comment:
Should also be a `debug` no?
```suggestion
        logger.info("Done fitting keras policy model")
```
```suggestion
        logger.info("Done fitting Keras Policy model.")
```